examples/contact-form-ai/workflows/AI_Contact_Form.yaml

```diff
@@ -0,0 +1,184 @@
+---
+active: false
+connections:
+  Email Response Structure:
+    ai_outputParser:
+      - - index: 0
+          node: Summary AI Agent
+          type: ai_outputParser
+  Groq Chat Model:
+    ai_languageModel:
+      - - index: 0
+          node: Summary AI Agent
+          type: ai_languageModel
+  Sample Data:
+    main:
+      - - index: 0
+          node: Summary AI Agent
+          type: main
+  Send Email:
+    main:
+      - - index: 0
+          node: Respond to Webhook
+          type: main
+  Summary AI Agent:
+    main:
+      - - index: 0
+          node: Send Email
+          type: main
+  Webhook:
+    main:
+      - - index: 0
+          node: Summary AI Agent
+          type: main
+  When clicking ‘Test workflow’:
+    main:
+      - - index: 0
+          node: Sample Data
+          type: main
+name: AI Contact Form
+nodes:
+  - id: 26476cd4-4435-4887-b419-ea2aaf703fd3
+    name: Webhook
+    parameters:
+      httpMethod: POST
+      options:
+        allowedOrigins: https://example.com
+        ignoreBots: false
+      path: ai-contact-form-submission
+      responseMode: responseNode
+    position:
+      - -2020
+      - -840
+    type: n8n-nodes-base.webhook
+    webhookId: e2103bd1-93dc-45b0-835a-50d87daa3ae1
+  - id: 776bb126-2715-4b6c-90c6-356624500d84
+    name: Respond to Webhook
+    parameters:
+      options:
+        responseCode: 200
+    position:
+      - -1200
+      - -940
+    type: n8n-nodes-base.respondToWebhook
+  - credentials:
+      smtp:
+        id: YkTaKDvPWacL6fvE
```

Comment:
TODO - I need to implement a way to fetch credentials references when running refresh. Or probably will keep it just as the only manual step to configure and set.